### ISSUE TITLE
Step 55 of OOP game should update instructions to mention where destructing should happen

### ISSUE THREAD

**Issue by jdwilkin4** [freeCodeCamp/freeCodeCamp#53201]
### Describe the Issue

A [camper](https://forum.freecodecamp.org/t/learn-intermediate-oop-by-building-a-platformer-game-step-55/664501) was trying to pass with this valid code here

```js
window.addEventListener("keydown", (event) => {
  const {key} = event;
});

```
but couldn't because the test is expecting this solution

```js
window.addEventListener("keydown", ({ key }) => {
  
});
```
we should update the directions to indicate where the destrucurting should happen. 

### Affected Page

https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures-v8/learn-intermediate-oop-by-building-a-platformer-game/step-55

### Your code

see explanation above

### Expected behavior

see explanation above

### System

- Device: [e.g. iPhone 6, Laptop]
- OS: [e.g. iOS 14, Windows 10, Ubuntu 20.04]
- Browser: [e.g. Chrome, Safari]
- Version: [e.g. 22]

**Comment by gikf** [comment]
Step 57 has similar pattern https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures-v8/learn-intermediate-oop-by-building-a-platformer-game/step-57.

Personally I'd reword instructions to indicate destructuring is expected in the parameter.

**Comment by RalphFilipek** [comment]
Thanks I was trying to solve it with AI something about 1,5h
